lnd/setup/index.js

```diff
@@ -73,17 +73,19 @@ async function channels(node) {
     if (!channels.length) {
       console.log("=> alice opening channels...");
       // open channels here
-      await asyncForEach(peersToMake, async (p) => {
-        await lightning.openChannel(node, {
-          pubkey: p.pubkey,
-          amount: 2000000,
-          push_amount: 1000000,
+      for (let i = 0; i < 2; i++) {
```

Comment:
instead of doing this maybe we can open a channel from carol -> bob or bob -> carol instead by editing the nodes.json file

feel like this approach is a bit clunky and then also we can test the amp payment splitting between two nodes

Reply:
`./lnd/setup/nodes.js` file

Reply:
and if you do this you'll probably have to call `await coinsAndChannels(nodes.nodes.bob);`

Reply:
yeah sounds good

Reply:
> and if you do this you'll probably have to call `await coinsAndChannels(nodes.nodes.bob);`

oops missed that, fixing now